.github/CONTRIBUTING.rst

```diff
@@ -0,0 +1,35 @@
+Contributing to the project
+===========================
+
+Contributions and issues are most welcome! All issues and pull requests are
+handled through GitHub_. Also, please check for any existing issues before
+filing a new one. If you have a great idea but it involves big changes, please
+file a ticket before making a pull request! We want to make sure you don't spend
+your time coding something that might not fit the scope of the project.
+
+.. _GitHub: https://github.com/DiamondLightSource/python3-pip-skeleton-cli/issues
+
+Issue or Discussion?
```

Comment:
Discussions is a nice idea. But we need to turn it on in the repo. 

We need a guide to configuring github repo and it should include this.